package.json

```diff
@@ -76,6 +76,11 @@
                     "dark": "./icons/refresh-dark.svg",
                     "light": "./icons/refresh-light.svg"
                 }
+            },
+            {
+                "category": "Bazel",
+                "command": "bazel.showOutput",
+                "title": "Show Output"
```

Comment:
I think a more descriptive title would be helpful here, like "Show Extension Log". "Show Output" sounds to me like Bazel stdout/stderr output.

Likewise, can you make the command name `bazel.showExtensionLog`?

Reply:
Done.